src/curvefit/uncertainty/draws.py

```diff
@@ -0,0 +1,140 @@
+import numpy as np
+from curvefit.utils.data import data_translator
+from curvefit.core.data import Data
+from curvefit.uncertainty.residual_model import _ResidualModel
+
+
+# TODO: add tests
+class Draws:
+    """
+        {begin_markdown _Draws}
+
+        {spell_markdown subclassed covs}
+
+        # `curvefit.uncertainty.draws.Draws`
+        ## A class for generating draws: predictions plus random residuals according to provided ResidualModel
+
+        This class simulates possible trajectories by adjusting out-of-sample part of the prediction
+        with expected residuals from ResidualModel. The user-controlled parameters are passed via init; the
+        parameters which need to be consistent to other parts of the pipeline (like `model`, `solver`,
+         `evaluation_space`, etc) are passed via create_draws(...).
+
+        ## Arguments
+        - `num_draws (int)`: the number of draws to take
```

Comment:
this is super picky but can you add a space before the bulleted lists? they won't render properly otherwise

Reply:
Sure! Fixed in this commit:
https://github.com/ihmeuw-msca/CurveFit/pull/85/commits/4bb91eba149fb21031a18a2bf598845a07ab15c4